x-pack/plugins/security_solution/server/lib/detection_engine/rules/utils.ts

```diff
@@ -316,7 +316,7 @@ export const legacyMigrate = async ({
   }
   /**
    * On update / patch I'm going to take the actions as they are, better off taking rules client.find (siem.notification) result
-   * and putting that into the actions array of the rule, then set the rules onThrottle property, notifyWhen and throttle from null -> actualy value (1hr etc..)
+   * and putting that into the actions array of the rule, then set the rules onThrottle property, notifyWhen and throttle from null -> actually value (1hr etc..)
```

Comment:
Super nit, but should this be "actual" instead of "actually"?

Reply:
Yep, will change it. Thanks